internal/tui/model_list.go

```diff
@@ -261,6 +382,12 @@ func newGetSuggestedBranch(description string, userBranches []project.Branch, re
 	}, nil
 }
 
+func removeSpecialChars(unsanitized string) string {
+	pattern := regexp.MustCompile(`[^A-Za-z\-_\/]+`)
```

Comment:
any particular reason we don't allow numbers? E.g. `seb/ticket-123` is a valid branch name, no?

Reply:
I was following the current logic 
![image](https://user-images.githubusercontent.com/2709271/233473501-a129a1ed-a95f-4c9a-ad40-fcd3a40bab48.png)
And we don't support numbers currently so I guest that was a mistake from the beginning I will fix it on both sides.